css/style.css

```diff
@@ -0,0 +1,44 @@
+@import url('https://fonts.googleapis.com/css?family=Libre+Baskerville');
+
+body {
+    background-color:#bfbfbf;
+    font-family: 'Libre Baskerville', serif;
+}
+.container{
+    background-color: #fff;
+}
+
+.img_nav{
+    width:50px;
+    height:50px;
+}
+
+.navbar-brand{
+    padding: 0;
+}
+
+.nav_tabs_flex{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.margin_right > .tabs{
+    margin-right: 40px;
+}
+ .main{
+    background-color: #fff;
+    margin-bottom:10px;
+    
+}
+
+.bg-gray{
+    background-color: #737373;
+    height: 200px;
+    color:white;
+}
+.bg-red{
+    background-color: red;
+    height: 3em;
```

Comment:
This height property is not needed, due to the line below specifying the height to 200px;